test/torchaudio_unittest/common_utils/gdrive_utils.py

```diff
@@ -0,0 +1,34 @@
+import requests
```

Comment:
We cannot use external resource in unit tests.
Also Google Drive throttle access. So it's unreliable.
What data is it?

Reply:
@mthrok It's the pre-trained model from the original HiFiGAN publication (https://github.com/jik876/hifi-gan#pretrained-model), they put the weights on Google Drive: https://drive.google.com/drive/folders/1-eEYTB5Av9jNql0WGBlRoi-WH2J7bp5Y

In the test I'm downloading them to check that our bundle has the same weights. Or do you think this test is redundant? Alternatively, we can upload the original weights to `download.pytorch.org` too.

Reply:
The output of HiFiGAN is waveform, we can use similar testing method in https://github.com/pytorch/audio/blob/main/test/integration_tests/source_separation_pipeline_test.py which calculate the Si-SNR score of estimated waveform and the clean reference.

Here we can use a clean reference waveform, generate the melspectrogram, and feed it to the pre-trained HiFiGAN to get the estimated waveform. We can compute Si-SNR score locally as the threshold for unit testing.

Reply:
If the goal is to ensure the parity with the original model, then we do not need the trained weights. Just port the original implementation as reference implementation in test, and feed synthetic data to check if the outputs are same.